sdk/healthinsights/azure-health-insights-cancerprofiling/CHANGELOG.md

```diff
@@ -0,0 +1,13 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
+
+- Azure AzureHealthInsights client library for Java. This package contains Microsoft Azure AzureHealthInsights client library.
```

Comment:
Please review the generated text and ensure it is of a good quality for end users.

Reply:
updated a bit as it is initial preview.
thanks